From 074dc48521c4efc780bd1b7a3e6fc43014c317ba Mon Sep 17 00:00:00 2001
From: Andrey Smirnov <andrew.smirnov@gmail.com>
Date: Fri, 2 Sep 2022 19:13:12 +0300
Subject: [PATCH 008/112] usb: dwc3: Hardcode jupiter ACPI device as extcon
 name

Hardcode jupiter ACPI device as extcon name in order to connect the
two together. This is really a hack and should really be coming from
our ACPI tables.

(cherry picked from commit 190ce1057f9bafd3ca4e11e7cda286c234666196)
Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>

[Fwd port to 6.0]
Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 drivers/usb/dwc3/dwc3-pci.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/dwc3/dwc3-pci.c b/drivers/usb/dwc3/dwc3-pci.c
index 6ecadc81b..9ce961c8b 100644
--- a/drivers/usb/dwc3/dwc3-pci.c
+++ b/drivers/usb/dwc3/dwc3-pci.c
@@ -195,6 +195,7 @@ static const struct property_entry dwc3_pci_mr_properties[] = {
 	PROPERTY_ENTRY_BOOL("usb-role-switch"),
 	PROPERTY_ENTRY_STRING("role-switch-default-mode", "host"),
 	PROPERTY_ENTRY_BOOL("linux,sysdev_is_parent"),
+	PROPERTY_ENTRY_STRING("linux,extcon-name", "VLV0100:00"),
 	{}
 };
 
-- 
2.52.0

